### PR TITLE
zh-cn for Scala Tour:Case Classes

### PR DESCRIPTION
**Pull request by declan94**
Simplified Chinese version of Scala Tour:Case Classes

### CODE REVIEW

File: _zh-cn/tour/case-classes.md [hunk 1]
```diff
@@ -13,3 +13,48 @@ language: zh-cn
...
+## 比较
+案例类在比较的时候是按值比较而非按引用比较：
+```
+case class Message(sender: String, recipient: String, body: String)
+
+val message2 = Message("jorge@catalonia.es", "guillaume@quebec.ca", "Com va?")
+val message3 = Message("jorge@catalonia.es", "guillaume@quebec.ca", "Com va?")
+val messagesAreTheSame = message2 == message3  // true
+```
+尽管`message2`和`message3`引用不同的对象，但是他们的值是相等的，所以`message2 == message3`为`true`。
```
Comment (jiminhsieh) [inline _zh-cn/tour/case-classes.md]: > 引用不同的对象

How about this? 參照到不同的物件

**Comment by hepin1989** [comment]
Are you in any QQ or wechat group?

**Comment by declan94** [comment]
> Are you in any QQ or wechat group?

Not yet

File: _zh-cn/tour/case-classes.md (hunk 1 above)
Comment (declan94) [reply _zh-cn/tour/case-classes.md]: Sorry but I don't think "參照到不同的物件" conforms to common programming terminologies in Chinese.

Comment (jiminhsieh) [reply _zh-cn/tour/case-classes.md]: Ah, I got it. It's just the different wording.

Comment (ctongfei) [reply _zh-cn/tour/case-classes.md]: 「对象」for "object" is mainland China usage, whereas in Taiwan it is usually 「物件」.

File: _zh-cn/tour/case-classes.md [hunk 2]
```diff
@@ -13,3 +13,48 @@ language: zh-cn
...
+你可以通过`copy`方法创建一个案例类实例的浅拷贝，同时可以指定构造参数来做一些改变。
+```
+case class Message(sender: String, recipient: String, body: String)
+val message4 = Message("julien@bretagne.fr", "travis@washington.us", "Me zo o komz gant ma amezeg")
+val message5 = message4.copy(sender = message4.recipient, recipient = "claire@bourgogne.fr")
+message5.sender  // travis@washington.us
+message5.recipient // claire@bourgogne.fr
+message5.body  // "Me zo o komz gant ma amezeg"
+```
+上述代码`message4`的`recipient`被用作为了`message5`的`sender`，重新指定了`message5`的`recipient`，而`message4`的`body`则是直接拷贝作为`message5`的`body`了。
```
Comment (dongxuwang) [inline _zh-cn/tour/case-classes.md]: > 被用作为了

这里不通顺

**Comment by SethTisue** [comment]
@declan94 want to get this across the finish line? looks like there's one review comment from @dongxuwang that you haven't responded to yet

File: _zh-cn/tour/case-classes.md [hunk 3]
```diff
@@ -13,3 +13,48 @@ language: zh-cn
 next-page: pattern-matching
 previous-page: multiple-parameter-lists
 ---
+
+案例类（Case classes）和普通类差不多，只有几点关键差别，接下来的介绍将会涵盖这些差别。案例类非常适合用于不可变的数据。下一节将会介绍他们在[模式匹配](pattern-matching.html)中的应用。
```
Comment (liufengyun) [inline _zh-cn/tour/case-classes.md]: 我看 [Programming Scala](https://book.douban.com/subject/27591387/) 中文版翻译为`样例类`。这里翻译为`案例类`有什么特别的缘由么？如果可能，最好能够统一。

我个人倾向`样例类`，因为字面上其与pattern match有所关联，而且不必创造新的词。但是，如果多数人共识使用`案例类`，我觉得也行。

Comment (sadhen) [reply _zh-cn/tour/case-classes.md]: 我们是处理成case类的

Comment (liufengyun) [reply _zh-cn/tour/case-classes.md]: `case类`我觉得也可以，没有任何误解的可能性。

**Comment by SethTisue** [comment]
@liufengyun is this ready for merge?

**Comment by SethTisue** [comment]
ping @liufengyun

**Comment by liufengyun** [comment]
Yes, I think it can go in.

**Comment by SethTisue** [comment]
thank you @declan94 and reviewers!

**Pull request merged**